src/views/Combine.vue

```diff
@@ -402,7 +387,34 @@ export default {
       // now that the new variant is selected we can update the path
       this.updatePath()
     },
-    async loadScenarioLayerSets (features) {
+    async loadScenarioLayerSetsByRoute () {
```

Comment:
Are there unit tests for these functions?

Reply:
Thanks, I will add some tests in a separate PR.